python/lsst/ctrl/bps/wms/panda/idds_tasks.py

```diff
@@ -160,12 +159,12 @@ def define_tasks(self):
                 == self.tasks_steps[task_name],
                 self.bps_workflow))
             bps_node = self.bps_workflow.get_job(picked_job_name)
-            task.queue = bps_node.compute_site
+            task.queue = bps_node.queue
```

Comment:
Just a note that this isn't using the GenericWorkflowJob.concurrency_limit.  The project should decide whether to use concurrency_limit and let the plugin decide if that limit is implemented as a queue or some other mechanism or if we should switch to just queues and then the plugin decide that a queue sometimes isn't a queue.  Or since this is a limited case just let the plugins be different.

Reply:
plugin can not easily propagate the concurrency limit into the queue configuration. Another question, is the concurrently limit specified as per submission level however expected to bind the whole payload active in the system, so this setting should be propagated into the queue configuration?

Reply:
The concurrency limit is a per GenericWorkflowJob setting.  Realistically, it is currently set for all jobs with a particular label (or final job).  The yaml allows it to be set for the entire workflow, but that doesn't really make sense.  This concurrency limit is supposed to throttle those jobs across all workflows.

The current design assumes that if a WMS or site is implementing this concurrency limit via queues that the plugin would internally do the conversion.  The plugin is allowed to have configuration that explains the conversion.   For example, there could be yaml:
```
pandaConcurrencyLimits:
    db_limit: <panda queue name>   # underscore in key db_limit because that's what the current value is in the job.
```

The yaml and code can be more complex where needed (e.g., different queues for different compute sites, maybe it isn't a directly translation to a queue name but must also use request_memory, etc.)

If folks want to change the design to either only allow queue-based implementations or have plugins translate queues to the other concurrency limit implementations, that's another direction we could go.